docker-compose.yml

```diff
@@ -1,32 +1,23 @@
 services:
-    db:
-        image: postgres
-        environment:
-            - POSTGRES_DB=postgres
-            - POSTGRES_USER=postgres
-            - POSTGRES_PASSWORD=postgres
-        volumes:
-            - .:/postgres_data/ # Persist data even if container shuts down volumes
-    web:
-        build: .
-        command: python manage.py runserver 0.0.0.0:8000
-        ports:
-            - "8000:8000"
-        depends_on:
-            - db
-            - migration
-        env_file:
-            - ./env_config/local.env
-        healthcheck:
-            test: curl -s -o /dev/null localhost:8000
-            interval: 3s
-            timeout: 5s
-            retries: 5
-    migration:
-        build: .
-        command: [ "bash", "-c", "while !</dev/tcp/db/5432; do sleep 1; done; python manage.py migrate" ]
-        depends_on:
-            - db
+  db:
+    extends:
+      file: ../docker-compose.base.yml
```

Comment:
This would be okay approach with monorepo, but not really with current wonky git submodules approach.
Maybe it'd be better to have whole `docker-compose.yml` in full stack repo, but then again - CI and testing would become more complex.